app/services/transfer_assignments_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class TransferAssignmentsService
+  attr_accessor :organization, :old_user, :new_user
+  def initialize(organization, old_user, new_user = nil)
+    @organization = organization
+    @old_user = old_user
+    @new_user = new_user || @organization.users.where.not(id: old_user.id).first
+  end
+
+  def transfer
+    return false unless user_owns_any_assignments?
+    all_assignments_of_user = organization.all_assignments.select do |assignment|
+      assignment.creator_id == old_user.id
+    end
+    all_assignments_of_user.map do |assignment|
```

Comment:
I'd group this all together to be a little more concise:

```ruby
organization.all_assignments.map do |assignment|
  next unless assignment.creator_id == old_user.id
  assignment.update(creator_id: new_user.id)
end
```

Reply:
done 👍🏼